src/chips/_chips.mixin.scss

```diff
@@ -0,0 +1,60 @@
+@import "../style/mixins";
+
+@mixin ssv-chips-variant($options: ()) {
+	$background-color: map-get($options, color);
+	$color: map-get($options, color-contrast);
+	$focus: map-get($options, focus-color);
+	$focused-greyscale: rgba($ssv-color-black, 0.2) !default;
+
+	&.ssv-chips {
+		.ssv-chips__item-image-container--static,
+		.ssv-chips__item-image-container--static .ssv-icon {
+			background-color: $background-color;
+			color: $color;
+		}
+
+		@include modifier("fill-none") {
+			.ssv-chips__item {
+				.ssv-chips__remove-item {
+					color: $ssv-color-black;
```

Comment:
ideally the black color is also passed as a variable, or at least mapped in function so it can be easily changed

Reply:
done